src/components/VideoDetail/VideoDetail.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DataContext from '../../context/DataContext';
+import VideoDetail from '.';
+
+describe('VideoDetail', () => {
+  it('must render VideoDetail and text related videos', () => {
+    const context = {
+      selectedVideo: [],
+      setSelectedVideo: jest.fn(),
+      relatedVideos: [],
+      favoriteVideos: [],
+      setFavoriteVideos: jest.fn(),
+    };
+    render(
+      <DataContext.Provider value={context}>
+        <VideoDetail />
+      </DataContext.Provider>
+    );
+    expect(screen.queryByText(/related videos/i)).toBeInTheDocument();
```

Comment:
```suggestion
    expect(screen.getByText(/related videos/i)).toBeInTheDocument();
```
`queryBy*` is used when you want to asset that something is not in the document and it can give you some false positives.

See: https://testing-library.com/docs/queries/about#types-of-queries and https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence